### PR TITLE
Ria 3601 notifications retry

### PR DESCRIPTION
**Pull request by jkumar-moj**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-3601

### Change description ###
 Notification failure confirmation message on errors

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/AdjournWithoutDateConfirmationTest.java [hunk 1]
```diff
@@ -34,9 +38,18 @@
 
     @Mock
     private Callback<AsylumCase> callback;
+    @Mock private CaseDetails<AsylumCase> caseDetails;
+    @Mock private AsylumCase asylumCase;
 
     private AdjournWithoutDateConfirmation handler = new AdjournWithoutDateConfirmation();
 
+    //@Before
```
Comment (makowalski) [inline src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/AdjournWithoutDateConfirmationTest.java:46]: we can remove commented part

**Review by makowalski — APPROVED** [review]
LGTM

**Review by Srihari-Nakkala — DISMISSED** [review]
LGTM

File: src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/RequestRespondentReviewConfirmationTest.java [hunk 2]
```diff
@@ -49,6 +64,36 @@ void should_return_confirmation() {
             .contains("Wait for the respondent");
     }
 
+    @Test
+    public void should_return_notification_failed_confirmation() {
```
Comment (Lee365) [inline src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/RequestRespondentReviewConfirmationTest.java:68]: Remove this 'public' modifier

**Review by Lee365 — CHANGES_REQUESTED** [review]

File: src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/RequestRespondentReviewConfirmationTest.java (hunk 2 above)
Comment (jkumar-moj) [reply src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/RequestRespondentReviewConfirmationTest.java]: Removed

**Review by Lee365 — APPROVED** [review]
LGTM

**Pull request merged**